htdocs/comm/action/card.php

```diff
@@ -1114,7 +1114,8 @@ function init_repeat()
 			$events[]=array('method' => 'getContacts', 'url' => dol_buildpath('/core/ajax/contacts.php?showempty=1',1), 'htmlname' => 'contactid', 'params' => array('add-customer-contact' => 'disabled'));
 			// TODO Refresh also list of project if $conf->global->PROJECT_ALLOW_TO_LINK_FROM_OTHER_COMPANY not defined with list linked to socid ?
 			// FIXME If we change company, we may get a project that does not match
-			print $form->select_company($object->socid, 'socid', '', 'SelectThirdParty', 1, 0, $events, 0, 'minwidth200');
+			print $form->select_thirdparty_list($object->socid, 'socid', '', 'SelectThirdParty', 1, 0, $events);
```

Comment:
Why changing this ?
The function select_company must be used in priority. This function follow the setup to decide if we load all database to show the page of just thirdparties after typing some keys.

select_thirdparty_list is already called bu select_company if setup "Wait you press a key before loading content of thirdparties combo list" is to "No".